src/bin/cratesfyi.rs

```diff
@@ -101,6 +101,8 @@ enum CommandLine {
     },
 
     StartRegistryWatcher {
+        #[arg(name = "SOCKET_ADDR", default_value = "0.0.0.0:3000")]
+        metric_server_socket_addr: String,
```

Comment:
You can use `SocketAddr` here to get clap to parse it and give a good error message.

Reply:
I didn't know about that, I updated the arguments & their usages.